rules/macos/privilege_escalation_explicit_creds_via_apple_scripting.toml

```diff
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2020/12/07"
+maturity = "production"
+updated_date = "2020/12/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of the security_authtrampoline process via Apple Scripting (osascript). This is a sign of
+execution with explicit logon credentials.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Execution with Explicit Credentials via Apple Scripting"
```

Comment:
```suggestion
name = "AppleScript Execution via Explicit Credentials"
```

Reply:
its the opposite, from an applescript it launch another program as another user (not necessarily root)